osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapCarousel.cs

```diff
@@ -118,6 +118,50 @@ void selectNextAndAssert(int amount)
             }
         }
 
+        [Test]
+        public void TestTraversalHoldDifficulties()
+        {
+            var sets = new List<BeatmapSetInfo>();
+
+            for (int i = 0; i < 20; i++)
+            {
+                var set = createTestBeatmapSet(i + 1);
+                sets.Add(set);
+            }
+
+            loadBeatmaps(sets);
+
+            void selectNextAndAssert(int amount, bool forwards, int expectedSet, int expectedDiff)
+            {
+                // Select very first or very last difficulty
+                setSelected(forwards ? 1 : 20, forwards ? 1 : 3);
+                string text = forwards ? "Next" : "Previous";
+                AddStep($"{text} difficulty {amount} times", () =>
+                {
+                    for (int i = 0; i < amount; i++)
+                    {
+                        carousel.SelectNext(forwards ? 1 : -1, false);
+                    }
+                });
+                waitForSelection(expectedSet, expectedDiff);
+            }
+
+            // Selects next set once, difficulty index doesn't change
+            selectNextAndAssert(3, true, 2, 1);
+            // Selects next set 16 times (50 // 3 == 16), difficulty index changes twice (50 % 3 == 2)
+            selectNextAndAssert(50, true, 17, 3);
+            // Travels around the carousel thrice (200/60 == 3)
```

Comment:
This should say 200 // 60

Reply:
where does "//" come from? i've never seen this used before

Reply:
It's floor division in programming languages that don't use // to start a comment

Reply:
seems like maybe `\` is more universal in the math world. or just use `/` since it's quite obvious here.